app/js/components/Visualizer.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const Visualizer = () => (
```

Comment:
nitpicky, but the better name for this would be `Visualization` (visualizers create visualizations)